tlvc/include/linuxmt/mm.h

```diff
@@ -27,6 +27,7 @@ typedef struct segment segment_s;
 #define SEG_FLAG_FDAT	 0x04   /* app fmemalloc far data */
 #define SEG_FLAG_EXTBUF	 0x05   /* ext/main memory buffers */
 #define SEG_FLAG_RAMDSK	 0x06   /* ram disk buffers */
+#define DEG_FLAG_BUFHEAD 0x07	/* bufheads for ext/xms buffers */
```

Comment:
Looks good. Don't forget to switch out SEG_FLAG_EXTBUF for this in fs/buffer.c in seg_alloc. I didn't see that in this PR so thought to mention it. I'll do the same in ELKS.